### PR TITLE
separate create_mock_data for creating mock data; added test for txfi…

### PR DESCRIPTION
**Pull request by dg1an3**
…eldpoint access

### CODE REVIEW

**Comment by dg1an3** [comment]
@SimonBiggs may be of use for issue #1391, though the mock txfield points are pretty lame

File: lib/pymedphys/tests/mosaiq/create_mock_data.py [hunk 1]
```diff
@@ -0,0 +1,241 @@
+# create mock patients
+from decimal import Decimal
+from struct import pack
+
+from pymedphys._imports import numpy as np
+from pymedphys._imports import pandas as pd
+from pymedphys._imports import pymssql, pytest, sqlalchemy
+from sqlalchemy.types import LargeBinary, Numeric
```
Comment (SimonBiggs) [inline lib/pymedphys/tests/mosaiq/create_mock_data.py:8]: Can we add `sqlalchemy.types` below the following line:

https://github.com/pymedphys/pymedphys/blob/ce939c05359b4cf1e57a0c866ec4e3cac905633a/lib/pymedphys/_imports/imports.py#L23

And then here, instead import as:

```suggestion
from pymedphys._imports import sqlalchemy
```

And then below, call `LargeBinary` and `Numeric` with `sqlalchemy.types.LargeBinary` and `sqlalchemy.types.Numeric`

File: lib/pymedphys/tests/mosaiq/create_mock_data.py [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+        dtype={
+            "Index": Numeric(precision=9, scale=3),
+            "A_Leaf_Set": LargeBinary(length=200),
+            "B_Leaf_Set": LargeBinary(length=200),
+            "Gantry_Ang": Numeric(precision=4, scale=1),
+            "Coll_Ang": Numeric(precision=4, scale=1),
+            "Coll_Y1": Numeric(precision=4, scale=1),
+            "Coll_Y2": Numeric(precision=4, scale=1),
+        },
+    )
```
Comment (SimonBiggs) [inline lib/pymedphys/tests/mosaiq/create_mock_data.py:239]: This is absolutely above and beyond! 🎉 Making my day 😄, thank you.

**Review by SimonBiggs — APPROVED** [review]
😄 !!! 🙂 LGTM.

**Pull request merged**